src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -942,7 +947,11 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.SetEnvironmentVariable.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.SetThreadErrorMode.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.SetFilePointerEx.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.SYSTEM_INFO.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.TimeZone.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.VirtualAlloc.cs" />
```

Comment:
We cannot have VirtualAlloc to be referenced in the .NET Native build, even if it is not actually called. VirtualAllloc is prohibited API in UWP apps. Referencing it will fail the static validation in the UWP build. Cou

I think ifdef inside `Interop.VirtualAlloc.cs` would be the easiest way to solve this. (And delete the ifdef `MemoryFailPoint.Windows.cs`.)

Reply:
Ah, the static validation and lack of CI bites me again. I will see what can I do.

Reply:
Sorry...

Reply:
I guess it's fine. I can compile the relevant bits with ENABLE_WINRT locally, but whatever post-build validation is normally performed I can't really do. I'm more concerned with wasting other people's valuable time on PR reviews when it's something I could probably figure myself from a build error.